ls1intum/Artemis
# `Programming exercises`: Fix task in problem statement with zero tests

### Checklist
#### General


#### Client

### Motivation and Context


Sometimes a task in a problem statement does not contain any tests. This can be the case if the exercise is only/partially graded manually, but the tasks are still used for structuring.
Currently such a task would still show "0 of 1 test passing".

### Description

This PR fixes this behavior such that "No tests" is displayed instead.
In case a hidden test is added to the task before the due date, the task will still show "0 of 1 test passing", this is out of scope for this PR.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Students
- 1 Programming Exercise

1. Log in to Artemis
2. Create a task in the problem statement without any tests or delete the tests from an existing task
3. Participate as a student and submit something
4. Look at the task and klick on the feedback to verify that there is no misleading text for this case

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage





### Screenshots


Before:
![Bildschirm­foto 2022-12-16 um 19 15 41](https://user-images.githubusercontent.com/38322605/208163033-03653f84-3968-44e7-ad44-24e9435eb9a0.png)
After:
![Bildschirm­foto 2022-12-16 um 19 16 24](https://user-images.githubusercontent.com/38322605/208163036-20801642-e16d-44cf-9352-1b77c1b2e434.png)
